crates/cargo-wdk/Cargo.toml

```diff
@@ -3,7 +3,7 @@ edition.workspace = true
 name = "cargo-wdk"
 version = "0.1.0"
 authors = ["Microsoft"]
-description = "A Cargo extension for building Windows drivers using the Windows Driver Kit (WDK)"
+description = "A development tool for Windows Rust drivers"
```

Comment:
Are you sure you want to omit that it's a cargo extension here, instead of changing it in the readme? Since the tool isn't invokable standalone as `wdk build` or `wdk new` afaik

Reply:
Changed it to: _A Cargo extension for developing Windows drivers in Rust_